services/src/main/java/io/scalecube/services/ServiceInfo.java

```diff
@@ -0,0 +1,79 @@
+package io.scalecube.services;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Helper class used to register service with tags as metadata in the scalecube cluster.
+ * parsing from service info to json and back.
+ */
+public class ServiceInfo {
+
+  @JsonIgnore
+  private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Specifically it can be such kind of code:

``` java
  String serviceTagPrefix = "myCoolService:tag:";
  List<Tag> serviceTags = metadata.entrySet().stream()
    .filter(e -> e.getKey().startsWith(serviceTagPrefix))
    .map(e -> new Tag(e.getKey().substring(serviceTagPrefix.length()), e.getValue()))
    .collect(Collectors.toList());
```